backend/coins/coin/account.go

```diff
@@ -0,0 +1,81 @@
+// Copyright 2018 Shift Devices AG
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package coin
+
+import (
+	"github.com/digitalbitbox/bitbox-wallet-app/backend/coins/coin/fee"
+	"github.com/digitalbitbox/bitbox-wallet-app/backend/signing"
+	"github.com/digitalbitbox/bitbox-wallet-app/util/observable"
+)
+
+// Account contains transfers of the given coin.
+type Account interface {
```

Comment:
Easier to step-by-step move the btc.Interface into a state where it is coin agnostic, and then just move it? What I did with my previous commits, so it is easy to follow.